### PR TITLE
fix[contracts]: remove upgrade cancellation functions

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Removes `cancelTransactionBundle` and `overrideTransactionBundle` functions from `L2ChugSplashDeployer.` Please see this issue for additional context: https://github.com/ethereum-optimism/optimism/issues/894

### CODE REVIEW

File: packages/contracts/contracts/chugsplash/L2ChugSplashDeployer.sol [hunk 1]
```diff
@@ -174,46 +174,6 @@ contract L2ChugSplashDeployer is Ownable {
...
-    function cancelTransactionBundle()
-        public
-        onlyOwner
-    {
-        require(
-            hasActiveBundle() == true,
-            "ChugSplashDeployer: cannot cancel when there is no active bundle"
-        );
-
-        emit BundleCancelled(
```
Comment (maurelian) [inline packages/contracts/contracts/chugsplash/L2ChugSplashDeployer.sol:189]: You should delete the event definition above too.

**Review by maurelian — CHANGES_REQUESTED** [review]
Having read the discussion at #894, this LGTM aside from one small change request.

File: packages/contracts/contracts/chugsplash/L2ChugSplashDeployer.sol (hunk 1 above)
Comment (smartcontracts) [reply packages/contracts/contracts/chugsplash/L2ChugSplashDeployer.sol:189]: Done!

**Comment by snario** [comment]
This needs to re-target `regenesis/0.4.0`.

**Review by ben-chain — APPROVED** [review]
LGTM, agree this is the way to go.

**Review by maurelian — APPROVED** [review]
LGTM

**Pull request merged**